interface/wx/systhemectrl.h

```diff
@@ -11,9 +11,9 @@
    A helper class making it possible to use system theme for any control.
 
    Under MSW, there an alternative theme available for the list and list-like
-   controls since Windows Vista. This theme us used by Windows Explorer list
-   and tree view and so is arguably more familiar to the users than the standard
-   appearance of these controls.
+   controls. This theme us used by Windows Explorer list and tree view and so
```

Comment:
```suggestion
   controls. This theme is used by Windows Explorer list and tree view and so 
```

There is also a missing "is" on line 13, which should read "Under MSW, there **is** an ...")

Reply:
Thanks, fixed locally (but not pushing yet to avoid rerunning the CI builds needlessly).